run_environment.sh

```diff
@@ -768,6 +774,25 @@ if [[ ${LIST_KEYS} == "true" ]]; then
     exit
 fi
 
+if [[ ${SYNC_MASTER} == "true" ]]; then
+        pushd ${AIRFLOW_BREEZE_AIRFLOW_DIR} || exit 1
+        set +e
+        git ls-remote --exit-code apache 2>/dev/null >/dev/null
+        if [[ $? != 0 ]]; then
+            echo "Adding remote apache ${APACHE_AIRFLOW_REPO_GIT}"
+            git remote add apache "${APACHE_AIRFLOW_REPO_GIT}"
+        fi
+        echo "Fetching apache repository"
+        git fetch apache
+        echo "Move local master to remote apache master"
```

Comment:
Maybe a bit more straightforward would be something like:
"Replace local master with remote apache master"

Reply:
I changed to "Force local master to be the same as remote apache master"